pid_error:
  MIN_DISTANCE: 0.1
  MAX_DISTANCE: 30.0
  CONTROL_DELAY_ESTIMATE: 0.5
  lookDistance: 2
  DESIRED_DISTANCE: 1
  currentMode: "left"
  currentEnumMode: 1
  currentGapAngle: 0

control:
  KP: 0.01
  KI: 0.000
  KD: 0.01
  N: 1
  K: 0.5
